example_config.json

```
@@ -0,0 +1,37 @@
+{
+    "debug": false,
+    "use_real_mongo": true,
```

I get that this is just an example config, but does it make more sense for this file to contain the default values?

```suggestion
    "use_real_mongo": false,
```